reframe/frontend/executors/policies.py

```diff
@@ -121,6 +117,7 @@ def __init__(self):
 
         self.task_listeners.append(self)
 
+
```

Comment:
Remove this line, it's against the coding style.

Reply:
Done.